### DISCUSSION TITLE
Will PocketBase survive?

### DISCUSSION THREAD

**Discussion by denisgurev** [pocketbase/pocketbase#3087] [Q&A] [answered]
Basically the title.

I tried to recommend pocketbase to my manager but he dismissed it because it is not v1 yet and it is developed by a single person.

Could you help me convince him otherwise?

**Comment by ganigeorgiev** [comment, marked as answer]
**I'm just some random developer that burns through his savings. I can't and don't want to make any promises.**

While bugs are fixed as soon as they are identified and I deliberately try to keep PocketBase scope fairly small to make it easier to maintain, your manager's concerns are valid and unfortunately I can't provide any solid arguments against them.

Like any other dependency, PocketBase is a tradeoff that you and your team have to decide whether the pros of using it will outweigh the cons.

No one likes when their code breaks with some dependency update and I fully understand that but at the same time it is very difficult and time consuming preventing it to happen (with Go even more) and the warnings in the README and the site docs are there for a reason.

Users nowadays just have too many expectations from open source (and even more from framework/toolkit projects) and because of this I sometimes regret publishing it before a stable release.

PocketBase is NOT something that was created after months of whiteboard thinking.
It is a non-commercial hobby project and as such it was created after work and over the weekends with a very narrow personal use-case in mind. Based on my experience with the maintenance of Presentator since ~2016, I honestly never expected PocketBase to become so much popular in such short amount of time.

I'm not fully satisfied with the current internals and some things will most likely break in the near future, but before finalizing and locking the APIs for further modifications I want to develop several applications with PocketBase to see and experience the various pain points and use cases on my own.

About the single developer concern - having a bus factor of 1 is not great, but I don't want to maintain a team with random/spare contributors, especially when there is no guarantee that they will stick after the merge to support it. Adding random strangers to a project is not helpful and it changes my focus to dealing with "people problems" (_in the end why some stranger should do for free the nitpicking changes I require?_) and I don't enjoy it.

**Reply by denisgurev** [reply]
As always thank you for the fast reply and honest answer. I really appreciate it!

**Reply by MuslemRahimi** [reply]
I contributed 5 Euros to support you brother. I apologize for the modest amount, as my current financial situation limits my ability to give more at this time.

I rely heavily on pocketbase for my small startup. I want to express my sincere gratitude for your dedicated efforts. Thank you very much for your hard work ❤️.

**Comment by whmeitzler** [comment]
Are you loosing money (savings) developing Pocketbase at this time? That shouldn't be. 
It may be, or will soon be, time for the community to start supporting your efforts directly. 
Either financially, or finding someone who can help you herd the cats if needed.

Your efforts on this project are exemplary, I've not seen [m]any OSS projects run so tightly by one guy. 
Pocketbase, IMHO, has a lot of potential to be the Rails or Django of Go- that's very valuable. 

This issue prompted me to throw a few dollars your way monthly, I hope others can do the same. 
Thanks for the work you do, Gani. We want you to be successful!
